extractors/python_functions/goodbye_extraction/code_snippet.md

```diff
@@ -1,7 +1,8 @@
 ```python
 import re
 
-YOUR_ATTRIBUTE = "your-text"
+YOUR_ATTRIBUTE: str = "text"
+YOUR_LABEL: str = "goodbye"
 
 def goodbye_extraction(record):
     regex = re.compile(r"((?:(good|Good)(?:[ ])?)?(bye|Bye)|Ciao|ciao|(?:(see you|See you)(?:[ ]?)(tomorrow|later|soon)?))")
```

Comment:
do we need to write somehwer that this is only working for a very limited set of values? e.g. no german goodbye

Reply:
Yes, either that or we need to expand our regex. For now I think it's okay for it to just be English